prowler/lib/mutelist/mutelist.py

```diff
@@ -303,7 +307,7 @@ def is_excepted(
             return False
 
     @staticmethod
-    def is_item_matched(matched_items, finding_items):
+    def is_item_matched(matched_items, finding_items, tag=False) -> bool:
```

Comment:
Can you update the method documentation?

Reply:
Sure